CSharp/Library/Microsoft.Bot.Connector.Shared/JwtConfig.cs

```diff
@@ -9,9 +9,12 @@
     public static class JwtConfig
     {
         /// <summary>
-        /// TO CHANNEL FROM BOT: Login URL
+        /// TO CHANNEL FROM BOT: Login URL format string. Bot developer can optionally specify
+        /// which tenant to obtain an access token from. The default tenant is "botframework.com".
+        /// Normally the chanels only accept tokens from the default tenant.

```

Comment:
chanels -> channels

"By default, the channel only accepts tokens from botframework.com".